client/hospital.lua

```diff
@@ -192,66 +192,53 @@ if config.useTarget then
 else
     CreateThread(function()
         for hospitalName, hospital in pairs(sharedConfig.locations.hospitals) do
-
             if hospital.checkIn then
-                local function enterCheckInZone()
-                    local numDoctors = lib.callback.await('qbx_ambulancejob:server:getNumDoctors')
-                    if numDoctors >= sharedConfig.minForCheckIn then
-                        lib.showTextUI(locale('text.call_doc'))
-                    else
-                        lib.showTextUI(locale('text.check_in'))
-                    end
-                end
-
-                local function outCheckInZone()
-                    lib.hideTextUI()
-                end
-
-                local function insideCheckInZone()
-                    if IsControlJustPressed(0, 38) then
-                        lib.hideTextUI()
-                        checkIn(hospitalName)
-                    end
-                end
-
                 lib.zones.box({
                     coords = hospital.checkIn,
                     size = vec3(2, 1, 2),
                     rotation = 18,
                     debug = config.debugPoly,
-                    onEnter = enterCheckInZone,
-                    onExit = outCheckInZone,
-                    inside = insideCheckInZone,
+                    onEnter = function()
+                        local numDoctors = lib.callback.await('qbx_ambulancejob:server:getNumDoctors')
+                        if numDoctors >= sharedConfig.minForCheckIn then
+                            lib.showTextUI(locale('text.call_doc'))
+                        else
+                            lib.showTextUI(locale('text.check_in'))
+                        end
+                    end,
```

Comment:
```suggestion
                    onEnter = function()
                        local doctorsAvailable = lib.callback.await('qbx_ambulancejob:server:getNumDoctors') >= sharedConfig.minForCheckIn
                        local label = doctorsAvailable and 'text.call_doc' or 'text.check_in'
                        lib.showTextUI(locale(label))
                    end,
```
Not sure if we want to leave that check here or if we need the server to return a number via that function